docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -26,6 +26,96 @@ To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/app
+
+The 128-bit UUID that is the base for UUIDs having 16-bit or 32-bit equivalents is defined available through the API in `BluetoothManager` as `BASE_UUID`.
+
+### Handling binary data
+The `Bytes` type, that aggregates all types in Bluetooth API used to pass binary data. It can be either a *byte[]* or a *DOMString* or a *Uint8Array*.
+
+`BluetoothManager` methods to manipulate binary data:
+   * `toByteArray()`
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toByteArray(dataInt8Array);
+      var second = tizen.bluetooth.toByteArray(dataUint8Array);
+      var third = tizen.bluetooth.toByteArray(dataString);
+      ```
+      Both first, second and third variables are equal arrays.
+
+   * `toDOMString()`
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toDOMString(dataInt8Array);
+      var second = tizen.bluetooth.toDOMString(dataUint8Array);
+      var third = tizen.bluetooth.toDOMString(dataString);
+      ```
+      Both first, second and third variables are equal to *"0x18b1"*.
+
+   * `toUint8Array()`
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toUint8Array(dataInt8Array);
+      var second = tizen.bluetooth.toUint8Array(dataUint8Array);
+      var third = tizen.bluetooth.toUint8Array(dataString);
+      ```
+      Both first, second and third variables are equal.
+
+   * `uuidTo128bit()`
+      ```
+      var uuidFrom16bits = tizen.bluetooth.uuidTo128bit("1234");
+      var uuidFrom32bits = tizen.bluetooth.uuidTo128bit("ab5690ef");
+      var uuidFrom128bits = tizen.bluetooth.uuidTo128bit("abcdef01-2345-6789-abcd-ef0123456789");
+      ```
+
+      uuidFrom16bits is equal to "00001234-0000-1000-8000-00805f9b34fb"  
+      uuidFrom32bits is equal to "ab5690ef-0000-1000-8000-00805f9b34fb"  
+      uuidFrom128bits is equal to "abcdef01-2345-6789-abcd-ef0123456789"
+
+   * `uuidToShortestPossible()`
+      ```
+      var from16Bit = tizen.bluetooth.uuidToShortestPossible("1234");
+      var from32Bit = tizen.bluetooth.uuidToShortestPossible("0000acdf");
+      var from128BitFirst = tizen.bluetooth.uuidToShortestPossible("ab5690ef-0000-1000-8000-00805F9B34FB");
+      var from128BitSecond = tizen.bluetooth.uuidToShortestPossible("abcdef01-2345-6789-abcd-ef0123456789");
+      ```
+
+      from16Bit is equal to "1234"  
+      from32Bit is equal to "acdf"  
+      from128BitFirst is equal to "ab5690ef"  
+      from128BitSecond is equal to "abcdef01-2345-6789-abcd-ef0123456789"
+
+   * `uuidsEqual()`
```

Comment:
This should go to the section about UUIDs.

Reply:
Done.